CenterForOpenScience/osf.io
# [PLAT-931] Retrieve/Delete User External Identities in APIv2

## Purpose

As part of out on going Emberification, we our exposing the external identity information to our V2 API.

## Changes

- Adds UserIdentitiesDetail view
- Adds UserIdentitiesList view
- Adds UserIdentitiesSerializer
- Adds tests

## QA Notes

- Go to a the identitites endpoint such as `http://localhost:8000/v2/users/<user_id>/settings/identities/` check to make sure it's cool
- Check the detail make sure it's cool too at `http://localhost:8000/v2/users/<user_id>/settings/identities/<identity_id>` like `http://localhost:8000/v2/users/<user_id>/settings/identities/ORCID/` is probably the only valid one for now.
- With the proper creds issue a DELETE against the `http://localhost:8000/v2/users/<user_id>/settings/identities/<identity_id>` make sure its is gone.

## Documentation

Docs will be add when this gets approved.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-931
